app/js/main.js

```diff
@@ -18,8 +18,24 @@ import {DemoPaywallController} from './demo-controller';
 import {MeteringDemo} from './metering';
 import {log} from './log';
 
+/* eslint-disable */
+const REGWALL = GaaMeteringRegwall;
```

Comment:
gTech and publishers use this file as an example, so I'd prefer we remove this alias since it adds a layer of abstraction